src/components/speaking-event/speaking-event.twig

```diff
@@ -0,0 +1,37 @@
+{% set _note %}{% block note %}{{ note }}{% endblock %}{% endset %}
+
+<div class="c-speaking-event{% if class %} {{class}}{% endif %}">
+  {% embed '@cloudfour/objects/media/media.twig' with {
+    reverse_markup: true,
+    datetime: datetime,
+    note: _note,
+    event: event,
+    location: location,
+    speakers: speakers,
+    href: href
+  } only %}
+    {% block object %}
+      {% include '@cloudfour/components/calendar-date/calendar-date.twig' with {
+        datetime: datetime,
+        note: note
+      } only %}
+    {% endblock %}
+    {% block content %}
+      <h3>
+        <a
+          href="{{ href }}"
+          class="c-speaking-event__link"
+        >
+          {{ event }}
+        </a>
+      </h3>
+      <p>{{ location }}</p>
+      <p>{% if speakers|length > 1 %}
+        Speakers:
+      {% else %}
+        Speaker:
+      {% endif %}
+      {{ speakers|join(', ') }} </p>
```

Comment:
We should consider wrapping these details in a `block`, which would allow the component to be used for events with different details where desired.

Reply:
Thanks for the suggestion, @tylersticka!

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1488/commits/8f8d13b4c36b6fb146dee383a861d31ce28f55c8